execution/execution.go

```diff
@@ -267,7 +261,10 @@ func newSubqueryFunction(e *parser.Call, t *parser.SubqueryExpr, storage *engsto
 	if parse.IsExtFunction(e.Func.Name) {
 		return nil, parse.ErrNotImplemented
 	}
-	// TODO: only instant queries for now.
+	// TODO: We dont pass arguments yet
+	if e.Func.Name == "quantile_over_time" {
```

Comment:
we dont yet pass arguments in subqueries so if we invoke quantile_over_time we get a panic since the scalar arguments of the function were not populated